src/fal/cli/model_generator/model_generator.py

```diff
@@ -99,6 +103,22 @@ def _check_path_safe_to_write(sql_path: Path, py_path: Path):
 CHECKSUM_REGEX = re.compile(r"FAL_GENERATED ([_\d\w]+)")
 
 
+def delete_generated_models(project_dir: str):
+    model_paths = _get_model_paths(project_dir)
+    for path in model_paths:
+        for root, dirs, files in os.walk(path):
```

Comment:
```suggestion
        for root, dirs, files in os.walk(path / GENERATED_DIR):   
```

right?

Reply:
That's a good question. Should we delete all fal generated files that might have been moved from the generated dir by mistake?